WordPress/Classes/Models/AbstractPost.m

```diff
@@ -633,4 +633,23 @@ - (void)markRemoteStatusFailed
     [self save];
 }
 
+- (NSString *)searchForImagesInContent
```

Comment:
I wonder if we could name this to something a little more descriptive (and possibly add a doc comment) – `URLForPrimaryContentImage`? `contentDisplayImageURL`?

Reply:
Or perhaps don't return a value from here, set `pathForDisplayImage` directly, and rename the method `updatePathForDisplayImage`. That might be best :)